src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/core-users-release.mdx

```diff
@@ -61,7 +51,19 @@ For a detailed comparison of capabilities, see [the capabilities table](/docs/ac
 This pricing version also changes [how billable users are calculated](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#user-calculation) and [rules on downgrading users](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/user-count-billing#downgrade-rules).
 
 
-## Details on opting in to this pricing version [#nr1core-optin]
+## Availability [#availability]
+
+The core users release is currently available to: 
+* New Relic organizations that were created on or after January 12, 2022. 
+* Organizations that existed before that date that meet certain requirements. If it's available to your organization, you'll receive messaging about that, either via [opt-in messaging](#nr1core-optin) in your billing management UI, or from your New Relic account representative.
```

Comment:
There's a prhase here that's a bit agentless where I think some "you" and "we" will help it sound friendlier. 

Instead of

```If it's available to your organization, you'll receive messaging about that, either via```

how about 

```If it's available to your organization, we'll let you know, either via```